File: Driver.java

public class Driver {

    private int driverId;
    private String driverName;
    private String phoneNumber;
    private String licenseNumber;
    private int experience;
    private String availabilityStatus;

    public Driver(int driverId, String driverName, String phoneNumber,
                String licenseNumber, int experience,
                String availabilityStatus) {

        this.driverId = driverId;
        this.driverName = driverName;
        this.phoneNumber = phoneNumber;
        this.licenseNumber = licenseNumber;
        this.experience = experience;
        this.availabilityStatus = availabilityStatus;
    }

    public void displayDriverDetails() {

        System.out.println("\n----------- Driver Details -----------");
        System.out.println("Driver ID : " + driverId);
        System.out.println("Driver Name : " + driverName);
        System.out.println("Phone Number : " + phoneNumber);
        System.out.println("License Number : " + licenseNumber);
        System.out.println("Experience : " + experience + " Years");
        System.out.println("Availability : " + availabilityStatus);
    }
}

File: c#/Driver.cs
public class Driver
{
    private int driverId;
    private string driverName;
    private string phoneNumber;
    private string licenseNumber;
    private int experience;
    private string availabilityStatus;

    public Driver(int driverId, string driverName, string phoneNumber,string licenseNumber, int experience,string availabilityStatus)
    {
        this.driverId = driverId;
        this.driverName = driverName;
        this.phoneNumber = phoneNumber;
        this.licenseNumber = licenseNumber;
        this.experience = experience;
        this.availabilityStatus = availabilityStatus;
    }

    public void displayDriverDetails()
    {
        Console.WriteLine("\n----------- Driver Details -----------");
        Console.WriteLine("Driver ID : " + driverId);
        Console.WriteLine("Driver Name : " + driverName);
        Console.WriteLine("Phone Number : " + phoneNumber);
        Console.WriteLine("License Number : " + licenseNumber);
        Console.WriteLine("Experience : " + experience + " Years");
        Console.WriteLine("Availability : " + availabilityStatus);
    }
}